e2e/cypress/integration/channel/react_to_last_message_shortcut_spec.js

```diff
@@ -0,0 +1,368 @@
+            });
+        });
+
+        cy.closeRHS();
+    });
+
+    it('Should open emoji picker for last post on center when RHS is open but focus is not on RHS textbox', () => {
+        // # Making sure there is at least a message to add comment to at center
+        cy.postMessage(MESSAGES.TINY);
+
+        // # Mouseover the entered post and click post comment icon.
+        cy.clickPostCommentIcon();
+
+        // * Check that the RHS is open
+        cy.get('#rhsContainer').should('be.visible');
+
+        // # Post a repliews in RHS.
+        cy.postMessageReplyInRHS(MESSAGES.SMALL);
+        cy.postMessageReplyInRHS(MESSAGES.TINY);
+
+        // # This post is in RHS, where reaction should not be added
+        cy.getLastPostId().then((lastPostId) => {
+            cy.get(`#${lastPostId}_message`).as('postInRHS');
+        });
+
+        // # Incoming posts from other user
+        cy.postMessageAs({sender: users['user-2'], message: MESSAGES.MEDIUM, channelId});
+
+        cy.wait(TIMEOUTS.SMALL);
+
+        // # Click anywhere to take focus away from RHS textbox
+        cy.get('#lhsList').within(() => {
+            cy.findByText('Town Square').click();
+        });
+
+        // # Emulate react to last message shortcut when focus is not on RHS or Center textbox
+        cy.get('body').type('{ctrl}{shift}\\');
+
+        // * Check that emoji picker is opened when focus is not on RHS or CENTER textbox
+        cy.get('#emojiPicker').should('exist');
+
+        // # Close the picker
+        cy.get('body').type('{esc}');
+        cy.wait(TIMEOUTS.TINY);
+
+        // # Focus back on Center textbox and enter shortcut
+        cy.get('#post_textbox', {timeout: TIMEOUTS.LARGE}).focus().type('{ctrl}{shift}\\');
+
+        // * Check that emoji picker is opened when focus is on CENTER textbox
+        cy.get('#emojiPicker').should('exist').within(() => {
+            // # Search for an emoji and add it to message
+            cy.findByPlaceholderText('Search Emoji').type('smile{enter}');
+        });
+
+        // # This post is in Center, where reaction be added
+        cy.getLastPostId().then((lastPostId) => {
+            cy.get(`#${lastPostId}_message`).as('postInCenter');
+        });
+
+        // * Check if the emoji picker has the entered reaction to the message in the center
+        cy.get('@postInCenter').within(() => {
+            cy.findByLabelText('reactions').should('exist');
+            cy.findByLabelText('remove reaction smile').should('exist');
+        });
+
+        // * Check if the emoji picker has not entered reaction to the message in the RHS
+        cy.get('@postInRHS').within(() => {
+            cy.findByLabelText('reactions').should('not.exist');
+            cy.findByLabelText('remove reaction smile').should('not.exist');
+        });
+
+        cy.closeRHS();
+    });
+
+    it('Should also open emoji picker for last message with Mac keyboard', () => {
```

Comment:
We can probably remove this case.
This will fail in case the test is run in platform other than Mac.  Currently our CI is on Linux but have plan to run in other platforms in the future.  This variation is now covered during local development - others are running on Mac, Linux and Windows.